api/src/utils/spatial-utils.ts

```diff
@@ -117,3 +119,37 @@ export function parseLatLongString(latLong: string): ILatLong | null {
 
   return { lat, long };
 }
+
+/*
+  Function to generate the SQL for insertion of a geometry collection
+*/
```

Comment:
nit: can you update the doc here (just so it uses the standard style with the '@params' etc.